src/Generation/FieldDetails.php

```diff
@@ -151,6 +151,9 @@ class FieldDetails
      * AFTER a package's 1.0 release, back to being optional.
      */
     private static $optionalToRequiredFixes = [
+        'google.cloud.bigquery.datatransfer.v1.EnrollDataSourcesRequest' => [],
+        'google.cloud.bigquery.datatransfer.v1.UnenrollDataSourcesRequest' => [],
+        'google.cloud.bigquery.datatransfer.v1.StartManualTransferRunsRequest' => [],
```

Comment:
Should there be fields in these lists?

Reply:
yep! Done.

Reply:
Sorry if I jumped on reviewing this too quickly 😛